l10n_nl_account_credit_control/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    'name': 'Credit Control for the Netherlands',
+    'summary': 'Dutch localization for credit control',
+    'license': 'AGPL-3',
+    'author': 'Onestein,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/l10n-netherlands',
+    'category': 'Localization',
+    'version': '11.0.1.0.0',
+    'development_status': 'Beta',
+    'depends': [
+        'account_credit_control',
```

Comment:
I assume you deliberately don't use https://github.com/OCA/account-financial-tools/tree/11.0/account_credit_control_dunning_fees ?

Reply:
Yes, correct.

Reply:
...I'm oldfashioned. Just tried out the github editor, seems to work

Reply:
I was already trying to do it, but it seems you already solved it thanks :smile:.